core-services/prow/02_config/_config.yaml

```diff
@@ -2364,6 +2364,7 @@ tide:
     repos:
     - openshift-priv/origin
     - openshift/origin
+    - openshift/network-tools
```

Comment:
this line and the one below in L2389, are they correct ? they got added automatically when I ran make new-repo.

Reply:
yes this is correct, its config for the merge automation